src/confcom/azext_confcom/template_util.py

```diff
@@ -275,6 +275,21 @@ def process_mounts(image_properties: dict, volumes: List[dict]) -> List[Dict[str
     return mounts
 
 
+def process_configmap(image_properties: dict) -> List[Dict[str, str]]:
```

Comment:
Can we update the samples for this addition?

Reply:
got it!